packages/arb-token-bridge-ui/src/components/TransferPanel/DepositCardL2Failure.tsx

```diff
@@ -21,9 +21,16 @@ export function DepositCardL2Failure({ tx }: { tx: MergedTransaction }) {
 
   return (
     <DepositCardContainer tx={tx}>
-      <span className="text-4xl font-semibold text-orange-dark">
+      <span className="text-4xl font-semibold text-brick-dark">
         {isRedeeming ? 'Re-executing...' : 'L2 transaction failed'}
       </span>
+
+      <div className="h-1" />
+      <span className="text-2xl font-normal text-brick-dark">
+        You have ~4 days to retry, so best to click the button now!
```

Comment:
This needs to be dynamic. The failed retryable tickets are kept for 7 days, so we should calculate it based on that and the time it was created.

Reply:
Done